stories/Canvas.stories.js

```diff
@@ -14,39 +14,51 @@ export default class CanvasExample extends React.Component {
     super(props);
     this.state = {
       layout: {
-        id: 'tklj94',
+        id: '0',
         type: 'element',
         tabs: [],
       },
     };
     this.onUpdate = this.onUpdate.bind(this);
+    this.onReset = this.onReset.bind(this);
+
+    if (props.persist) {
+      const state = JSON.parse(localStorage.getItem('state'));
```

Comment:
what if 'state' key is undefined?

Reply:
It's can be possible in case when Irina found some bug during the testing, and then she can press button `reset` which will reset local storage and reload the page)

It's just an example of using canvas with some persistor, the implementation of data persistent not related to the canvas.